includes/widgets/class-wc-widget-layered-nav.php

```diff
@@ -452,7 +452,7 @@ protected function layered_nav_list( $terms, $taxonomy, $query_type ) {
 
 				// Add Query type Arg to URL.
 				if ( 'or' === $query_type && ! ( 1 === count( $current_filter ) && $option_is_set ) ) {
-					$link = add_query_arg( 'query_type_' . sanitize_title( str_replace( 'pa_', '', $taxonomy ) ), 'or', $link );
```

Comment:
Can you see a reason `sanitize_title` was used here and in the layered nav filter widger previously? Possibly special chars?

Reply:
The only reason I can see is that attributes/values can be the local ones and it was added as a precaution.